.github/workflows/check-flake-files.yml

```diff
@@ -0,0 +1,55 @@
+name: "Check Flake Groups"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'flakes/**.toml'
+
+jobs:
+  automatic-custom-flakes-check:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+
+    env:
+      RUST_LOG: debug
+    
+    steps:
+
+    - name: Checking out the repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Installing Nix
+      uses: cachix/install-nix-action@v16
+      with:
+        install_url: https://nixos-nix-install-tests.cachix.org/serve/i6laym9jw3wg9mw6ncyrk6gjx4l34vvx/install
+        install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
+        extra_nix_config: |
+          experimental-features = nix-command flakes
+          access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need any of this? Latest nix should be fine, and flakes are enabled by default

Reply:
Never mind, fixed in #433